# -*- mode: snippet -*-
# name: catch backend statement
# key: cc
# --
.catch((err: Error) => {
    this.log.error('$1 failed', err);
    throw new Error('internalServerError');
});
$0